src/main/resources/swagger.api/ecs-tlr.yaml

```diff
@@ -0,0 +1,85 @@
+openapi: 3.0.0
+info:
+  title: ECS TLR API
+  version: v1
+paths:
+  /tlr/ecs-tlr:
+    post:
+      description: Create ECS TLR
+      operationId: postEcsTlr
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ecs-tlr"
+        required: true
+      responses:
+        '201':
+          $ref: "#/components/responses/ecs-tlr"
+        '400':
```

Comment:
Our controller does not return 400 (yet?)

Reply:
Our controller currently does not directly return a 400 response, however, the Swagger schema handles the validation and returns a 400 response if the incoming request is deemed invalid